README.md

```diff
@@ -690,11 +690,11 @@ If you're using the `X-SourceMap` header instead, you can just omit `sourceMap.u
   `foo` is certain to not throw, i.e. not `null` or `undefined`.
 
 - `reduce_funcs` (default: `true`) -- Allows single-use functions to be
-  inlined as function expressions when permissible allowing further 
-  optimization.  Enabled by default.  Option depends on `reduce_vars` 
-  being enabled.  Some code runs faster in the Chrome V8 engine if this 
+  inlined as function expressions when permissible allowing further
+  optimization.  Enabled by default.  Option depends on `reduce_vars`
+  being enabled.  Some code runs faster in the Chrome V8 engine if this
   option is disabled.  Does not negatively impact other major browsers.
-  
+
```

Comment:
The github builtin editor appears to add those trailing spaces.

Reply:
Not to worry - VSCode will pick up the slack :wink: